optimize/backend/src/test/java/io/camunda/optimize/service/util/DmnModelUtilTest.java

```diff
@@ -0,0 +1,127 @@
+import org.camunda.bpm.model.dmn.DmnModelInstance;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+public class DmnModelUtilTest {
+
+  private static final String DMN_V1 =
+      """
+      <?xml version="1.0" encoding="UTF-8"?>
+      <definitions xmlns="https://www.omg.org/spec/DMN/20191111/MODEL/"
+         xmlns:dmndi="https://www.omg.org/spec/DMN/20191111/DMNDI/"
+         xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/"
+         xmlns:biodi="http://bpmn.io/schema/dmn/biodi/2.0"
+         id="force_users" name="Force Users"
+         namespace="http://camunda.org/schema/1.0/dmn" exporter="Camunda Modeler"
+         exporterVersion="4.11.0">
+        <decision id="jedi_or_sith" name="Jedi or Sith">
+          <decisionTable id="DecisionTable_14n3bxx">
+            <input id="Input_1" label="Lightsaber color" biodi:width="192">
+              <inputExpression id="InputExpression_1" typeRef="string">
+                <text>lightsaberColor</text>
+              </inputExpression>
+            </input>
+            <output id="Output_1" label="Jedi or Sith" name="jedi_or_sith">
+              <outputValues id="UnaryTests_0hj346a">
+                <text>"Jedi","Sith"</text>
+              </outputValues>
+            </output>
+            <rule id="DecisionRule_0zumznl">
+              <inputEntry id="UnaryTests_0leuxqi">
+                <text>"blue"</text>
+              </inputEntry>
+              <outputEntry id="LiteralExpression_0c9vpz8">
+                <text>"Jedi"</text>
+              </outputEntry>
+            </rule>
+          </decisionTable>
+        </decision>
+      </definitions>
+      """;
+
+  @Test
+  void parseDmnModelShouldReturnModelInstanceWhenValidXml() {
+    // Given: A valid DMN XML string
+
+    // When: parseDmnModel is called
+    var result = DmnModelUtil.parseDmnModel(DMN_V1);
+
+    // Then: A valid DmnModelInstance should be returned
+    assertThat(result).isNotNull();
+  }
+
+  @Test
+  void parseDmnModelShouldThrowOptimizeRuntimeExceptionWhenInvalidXml() {
+    // Given: An invalid DMN XML string
+    String dmnXml = "<invalid></invalid>";
+
+    // When & Then: parseDmnModel should throw OptimizeRuntimeException
+    assertThatThrownBy(() -> DmnModelUtil.parseDmnModel(dmnXml))
+        .isInstanceOf(DmnModelException.class)
+        .hasMessage("Unable to parse model");
+  }
+
+  @Test
+  void extractDecisionDefinitionNameShouldReturnNameWhenDecisionExists() {
+    // Given: A valid DMN XML string and a decision key
+    String decisionKey = "jedi_or_sith";
+
+    // When: extractDecisionDefinitionName is called
+    Optional<String> result = DmnModelUtil.extractDecisionDefinitionName(decisionKey, DMN_V1);
+
+    // Then: The name of the decision should be returned
+    assertThat(result).isPresent().get().isEqualTo("Jedi or Sith");
+  }
+
+  @Test
+  void extractDecisionDefinitionNameShouldReturnEmptyWhenDecisionNotFound() {
```

Comment:
```suggestion
  void shouldReturnEmptyWhenDecisionKeyNotFound() {
```

Notice the format (`should...`). Please apply universally for all test names you write

Reply:
fixed